CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 4.14.0
+
+- Allow use of multiple GA customDimensionIndex.
+- Configurable duration (in days) for AB Test cookie.
+- Add assignment of jQuery explicitly so it's consistent with other modules.
+- Allow base scripts to run within a module loader.
+- Force push tracking branch on new version.
```

Comment:
Can probably remove `Add assignment of jQuery explicitly so it's consistent with other modules.` as well as that was just an ancillary piece of work.